src/sections/Meshery/Meshery-integrations/Integration.style.js

```diff
@@ -191,3 +189,64 @@ export const IntegrationCard = styled.div`
     }
   }
 `;
+
+export const IntegrationSlider = styled(Slider)`
+  .slick-prev:hover, .slick-next:hover{
+    box-shadow: none;
+    outline:none;
+  }
+
+  .slick-arrow {
+    margin-top: 1.4rem;
+    width: 2rem;
+    height: 4rem;
+  }
+
+  .slick-disabled {
+    pointer-events: none;
+    opacity: 0.5;
+  }
+
+  .slick-arrow:before {
+    color: ${(props) => props.theme.DarkTheme ? props.theme.white : props.theme.black};
+    font-size: 4.5rem;
+    display: inline-block;
+  }
+
+  .slick-arrow:hover:before {
+    color: #00b39f;
+  }
+
+  .slick-slide {
+    width: auto !important;
+    margin: 0 .5rem;
+  }
+
+  .slick-next{
+    right: -2.5rem;
+  }
+
+  .slick-prev{
+    left: -2.5rem;
+  }
+
+  .slick-prev:before {
+    content:"‹";
+    line-height: 0;
+    opacity: 1;
+  }
+
+  .slick-track {
+    display: flex;
+  }
+
+  .slick-next:before {
+    content: "›";
+    line-height: 0;
+    opacity: 1;
+  }
+
+  .slick-prev, .slick-next {
+    top:0
```

Comment:
We would have to update it to `0.25rem` because the arrows don't seem to be vertically center aligned as of now. @asubedy please confirm it in Safari as well